libgalois/test/property-index.cpp

```diff
@@ -21,12 +26,7 @@ using Edge = NodeOrEdge<katana::GraphTopology::Edge>;
 
 template <>
 katana::Result<katana::PropertyIndex<katana::GraphTopology::Node>*>
```

Comment:
You need these empty template declarations to make the `NodeOrEdge` trick work?  They can't just be overloads?